SimTracker/TrackerHitAssociation/plugins/SimDoubletsProducer.cc

```diff
@@ -0,0 +1,271 @@
+// -*- C++ -*-
+//
+// Package:    SimTracker/TrackerHitAssociation
+// Class:      SimDoubletsProducer
+//
+
+// user include files
+#include "SimTracker/TrackerHitAssociation/plugins/SimDoubletsProducer.h"
```

Comment:
Plugins are preferred to be kept in one file (i.e. not split to separate header and a source files, see 6.7 in https://cms-sw.github.io/cms_coding_rules.html)